### PR TITLE
[ENH]: Add embedded gui for beckhoff axis with custom eps screen

### PR DESCRIPTION
**Pull request by jozamudi**
## Description

Added 'embedded' ui file for motors that have custom eps screens.

## Motivation and Context

Adding the 'embedded' file allows for typhos screens to open using the compact controls.

## Where Has This Been Documented?

image

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1204-Add_embedded_gui_for_beckhoff_axis_with_custom_eps_screen.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+Adding the 'embedded' file allows for typhos screens to open using the compact
+controls.
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1204-Add_embedded_gui_for_beckhoff_axis_with_custom_eps_screen.rst:11]: nitpick: bullet point formatting, so the release script doesn't skip this later
```suggestion
- Adding the 'embedded' file allows for typhos screens to open using the compact
  controls.
```

File: docs/source/upcoming_release_notes/1204-Add_embedded_gui_for_beckhoff_axis_with_custom_eps_screen.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+jozamudi
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1204-Add_embedded_gui_for_beckhoff_axis_with_custom_eps_screen.rst:31]: nitpick: formatting
```suggestion
- jozamudi
```

File: pcdsdevices/ui/BeckhoffAxisEPSCustom.embedded.ui [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>EPS:</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="PyDMShellCommand" name="PyDMShellCommand">
```
Comment (ZLLentz) [inline pcdsdevices/ui/BeckhoffAxisEPSCustom.embedded.ui:97]: This is implemented as a `PyDMShellCommand` widget that shells out to `pydm`.
Is this preferable to using a `PyDMRelatedDisplayButton` widget that opens a new window in the same process?

**Review by ZLLentz — COMMENTED** [review]
Some minor nitpicks and a question
Overall this is a great addition

File: pcdsdevices/ui/BeckhoffAxisEPSCustom.embedded.ui [hunk 4]
```diff
@@ -125,35 +109,27 @@
...
-       </property>
-       <property name="commands" stdset="0">
+       <property name="macros" stdset="0">
         <stringlist>
-         <string>pydm --hide-nav-bar --hide-menu-bar --hide-status-bar /cds/group/pcds/epics-dev/screens/pydm/eps_screens/${beamline}/${name}.ui</string>
+         <string>{}</string>
         </stringlist>
        </property>
+       <property name="openInNewWindow" stdset="0">
+        <bool>false</bool>
```
Comment (ZLLentz) [inline pcdsdevices/ui/BeckhoffAxisEPSCustom.embedded.ui:131]: You probably want this to open in a new window, right?

**Comment by jozamudi** [comment]
Sorry for the late follow up, I made the suggested changes.

**Pull request merged**